MaximumValue.c

```diff
@@ -0,0 +1,20 @@
+// program that prompts the user to enter a list of integers separated by commas and then finds the maximum value in the list.
+ #include <stdio.h>
+ int main() {
+   int max = 0;
+   char input[100];
+   printf("Enter a list of integers separated by commas: ");
+   scanf("%[^\n]", input);  
+   for (int i = 0; input[i] != '\0'; i++) {
+     int num = 0;
+     while (input[i] >= '0' && input[i] <= '9') {
+        num = num * 10 + (input[i] - '0');
+        i++;
+     }
+     if (num > max) {
```

Comment:
Take care of negative numbers. 
Ex: 34, 67657, -564655, +45346
Expected output should be 67657
Actual is 564655

Reply:
updated